cpp/inc/bond/ext/grpc/detail/client_call_data.h

```diff
@@ -71,8 +71,8 @@ struct client_unary_call_data
           _threadPool(std::move(threadPool)),
           _responseReader(),
           _callbackArgs(context),
-          _self(),
-          _cb(cb)
+          _cb(cb),
```

Comment:
does the order matter here?

Reply:
Members are initialized in the order they are declared in. I like to keep the initializer list in the same order. It reduces cognitive burden, and some compilers warn if the orders differ.